shared/src/iosMain/kotlin/co/touchlab/droidcon/util/BundleResourceReader.kt

```diff
@@ -33,7 +33,11 @@ class BundleResourceReader(
                 name to null
             }
         }
-        val path = bundle.pathForResource(filename, type) ?: error("Couldn't get path of $name (parsed as: ${listOfNotNull(filename, type).joinToString(".")})")
+        val path = bundle
+            .pathForResource(filename, type) ?: error(
+            "Couldn't get path of $name (parsed as: ${listOfNotNull(filename, type)
+                .joinToString(".")})"
+        )
```

Comment:
These lines seem to be weirdly formatted. I'd expect something like:

```suggestion
        val path = bundle.pathForResource(filename, type) ?: error(
            "Couldn't get path of $name (parsed as: ${listOfNotNull(filename, type)
                .joinToString(".")})"
        )
```